src/seedu/addressbook/AddressBook.java

```diff
@@ -29,1158 +30,1260 @@
  * This class is used to maintain a list of person data which are saved
  * in a text file.
  **/
```

Comment:
Commit after every time you do a major change such refactoring a piece of code with an appropriate commit message.  Here you have replaced a big blob of code. Hard to see what changes have gone in.